src/graph/entries.mo

```diff
@@ -0,0 +1,45 @@
+import Array "mo:stdlib/array.mo";
+import Blob "mo:stdlib/blob.mo";
+import HashMap "mo:stdlib/hashMap.mo";
+import Hash "mo:stdlib/hash.mo";
+import Iter "mo:stdlib/iter.mo";
+import List "mo:stdlib/list.mo";
+import Nat "mo:stdlib/nat.mo";
+import Option "mo:stdlib/option.mo";
+
+import Types "./types.mo";
```

Comment:
Drop `.mo` extension from all imports (this shouldn't even work, see [issue](https://github.com/dfinity-lab/motoko/issues/1127)).